### DISCUSSION TITLE
My email has been reclaimed, do the contributions I made with it still belong to me?

### DISCUSSION THREAD

**Discussion by lhpqaq** [community/community#137459] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My email has been reclaimed, do the contributions I made with it still belong to me?
If someone binds my old email, does my contribution belong to them? 
Thanks

**Comment by phuyalgaurav** [comment, marked as answer]
Your contributions remain yours even if your email is reclaimed. Someone binding your old email won't gain ownership of your contributions. 👍
